overview.md

```diff
@@ -40,6 +40,6 @@ This information cannot directly identify any individual.
 
 You must acknowledge that you have read the VMware CEIP policy before you can proceed with the
 installation.
-For more information, see [Configure the installation settings](install.md#configure-installation-settings) in _Install Application Service Adapter_.
-To opt out of telemetry participation after installation, see
-[Opting out of telemetry reporting](install.md#opt-out-telemetry) in _Install Application Service Adapter_.
+For more information, see [Tanzu Application Platform Telemetry](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.5/tap/telemetry-overview.html).
+To opt out of telemetry collection after installation, see
```

Comment:
I think we should work the word "participation" in here still somehow. That's how other docs refer to the CEIP program. E.g. https://docs.vmware.com/en/VMware-Tanzu-Kubernetes-Grid/1.6/vmware-tanzu-kubernetes-grid-16/GUID-cluster-lifecycle-ceip.html

```suggestion
To opt out of CEIP participation and telemetry collection after installation, see
```

Reply:
Or maybe the "and telemetry collection" bit in my suggestion is redundant with the name of the link we direct them at. So it could just be "To opt out of CEIP participation after installation, see"

Reply:
Updated